File: go/fizbuz.go
package main

import "fmt"

func main() {
  for i := 1; i <= 100; i++ {
    if i % 15 == 0 {
      fmt.Println("FizBuz")
    } else if i % 3 == 0 {
      fmt.Println("Fiz")
    } else if i % 5 == 0 {
      fmt.Println("Buz")
    } else {
      fmt.Println(i)
    }
  }
}

File: rust/fizbuz.rs
fn main() {
  for i in range(1, 101) {
    match (i % 3, i % 5) {
      (0, 0) => println!("FizBuz"),
      (0, _) => println!("Fiz"),
      (_, 0) => println!("Buz"),
      _ => println!("{}", i)
    }
  }
}
